src/demo/LeafletMapDemo.js

```diff
@@ -6,29 +6,11 @@ import LeafletMap from "../lib/components/LeafletMap";
 // Assets
 import exampleData from "./example-data/leaflet-map.json";
 
-const DEFAULT_COLORMAP = {
-    colors: [
-        "#0d0887",
-        "#46039f",
-        "#7201a8",
-        "#9c179e",
-        "#bd3786",
-        "#d8576b",
-        "#ed7953",
-        "#fb9f3a",
-        "#fdca26",
-        "#f0f921",
-    ],
-    prefixZeroAlpha: false,
-    scaleType: "linear",
-    cutPointMin: 3400,
-    cutPointMax: 3500,
-};
-
 const LeafletMapDemo = () => {
     const [layers, setLayers] = useState(exampleData.layers.slice(1, 5));
```

Comment:
The code is useState is executed each render (even though it's assigned once to the state vars). Using a callback instead is a minor optimisation to prevent the execution
```suggestion
    const [layers, setLayers] = useState(() => exampleData.layers.slice(1, 5));
```